config/webpack.common.js

```diff
@@ -102,6 +102,7 @@ module.exports = {
       "window.Tether": "tether"
     })
   ],
+  postcss: () => [autoprefixer({browsers: ['last 2 versions']})],
```

Comment:
Totally doesn't apply here, but I didn't know you could target your own sites usage data using browserlist. That's a potentially neat feature.

https://github.com/ai/browserslist#custom-usage-data